media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -19,7 +19,8 @@ HOMEPAGE="https://pipewire.org/"
 
 LICENSE="LGPL-2.1+"
 SLOT="0/0.3"
-IUSE="aac aptx bluetooth doc extra gstreamer jack-client ldac pipewire-alsa systemd test v4l"
+IUSE="aac aptx bluetooth doc extra gstreamer jack-client jack-sdk ldac pipewire-alsa systemd test
+v4l"
```

Comment:
Keep a long line or break into multiple lines with
```bash
IUSE="aac ...."
IUSE+=" v4l"
```

Reply:
Alright, will do the former here.